test/lib/ansible_test/_data/sanity/validate-modules/validate_modules/main.py

```diff
@@ -1450,6 +1468,72 @@ def _validate_argument_spec(self, docs, spec, kwargs, context=None, last_context
+                else:
+                    compare_version = LOOSE_ANSIBLE_VERSION
+                    version_of_what = "Ansible (%s)" % ansible_version
+                    version_parser_error = "the version number cannot be parsed"
+
+                removed_in_version = data.get('removed_in_version', None)
+                if removed_in_version is not None:
+                    try:
+                        if compare_version >= self.Version(str(removed_in_version)):
+                            msg = "Argument '%s' in argument_spec" % arg
+                            if context:
+                                msg += " found in %s" % " -> ".join(context)
+                            msg += " has a deprecated removed_in_version '%s'," % removed_in_version
+                            msg += " i.e. the version is less than or equal to the current version of %s" % version_of_what
+                            self.reporter.error(
+                                path=self.object_path,
+                                code='ansible-deprecated-version',
+                                msg=msg,
+                            )
+                    except ValueError as dummy:
+                        msg = "Argument '%s' in argument_spec" % arg
+                        if context:
+                            msg += " found in %s" % " -> ".join(context)
+                        msg += " has an invalid removed_in_version '%s'," % removed_in_version
+                        msg += " i.e. %s" % version_parser_error
+                        self.reporter.error(
+                            path=self.object_path,
+                            code='ansible-invalid-version',
+                            msg=msg,
+                        )
+
+                deprecated_aliases = data.get('deprecated_aliases', None)
+                if deprecated_aliases is not None:
+                    for deprecated_alias in deprecated_aliases:
+                        try:
+                            if compare_version >= self.Version(str(deprecated_alias['version'])):
+                                msg = "Argument '%s' in argument_spec" % arg
+                                if context:
+                                    msg += " found in %s" % " -> ".join(context)
+                                msg += " has deprecated aliases '%s' with removal in version '%s'," % (
+                                    deprecated_alias['name'], deprecated_alias['version'])
+                                msg += " i.e. the version is less than or equal to the current version of %s" % version_of_what
+                                self.reporter.error(
+                                    path=self.object_path,
+                                    code='ansible-deprecated-version',
+                                    msg=msg,
+                                )
+                        except ValueError as dummy:
```

Comment:
Since `dummy` isn't used don't include it:

```suggestion
                    except ValueError:
```

Reply:
Fixed both.